### DISCUSSION TITLE
Wt32-eth01 bridge

### DISCUSSION THREAD

**Discussion by BlackOrca** [arendst/Tasmota#19365] [Q&A]
Hi,
I have an wt32-eth01 flashed with tasmota. Wifi is connected to a solar Inverter and eth is connected to my LAN. It is possibile to bridge the communication between eth and wifi?
I want to access from LAN side a IP and Port on the wifi side. A Port forwarding as a simple solution.
Br,
Dennis

**Comment by barbudor** [comment]
There is a range-extender feature that I've heard is working in wifi-to-wifi where the ESP32 acts as an AP for other Tasmotas (and may be other devices) and connect to another "WAN" by wifi. It may also work with a Ethernet based "WAN" ?
Check https://tasmota.github.io/docs/Range-Extender/
There have been some discussions here about that feature

**Reply by joba-1** [reply]
> (and may be other devices)

definitely :)
But I also have zero XP with WIFI to LAN.

**Comment by Jason2866** [comment]
The ""simple" port forwarding needs to be implemented. It is not plain easy.

**Comment by BlackOrca** [comment]
Thank you for your Assessment!

**Comment by matze20211** [comment]
Hi, have the same problem, wt32-eth0 with no wifi connection, connected via cable and used as a AP for some tasmota devices. rangeextender is only useable, if there is a wifi connection.

is there a solution?

matze20211

**Reply by sfromis** [reply]
That is just not an use case for the [Range Extender](https://tasmota.github.io/docs/Range-Extender/) 
> This feature allows Tasmota to publish an AP with a dedicated SSID that is then routed to the core WiFi network.

Thus, there is no "problem"; merely a wish for an enhancement of a feature. And with open source, the next question is who's "feeling inspired" to do the work, if anyone.
